stingray/pulse/search.py

```diff
@@ -129,13 +149,17 @@ def z_n_search(times, frequencies, nharm=4, nbin=128, segment_size=5000,
         length of the good time intervals.)
```

Comment:
Same: please add new keywords to docstrings (maybe just check all of the functions with new parameters?)

Reply:
Also done